src/instructlab/server.py

```diff
@@ -163,6 +165,12 @@ def server(
     queue=None,
 ):
     """Start OpenAI-compatible server"""
+    if not Path(model_path).is_file():
+        logger.error(
+            f"Model file does not exist: '{model_path}'. If the model doesn't exist, download it using the 'ilab download' command. Please verify the model path in 'config.yaml' or '--model-path'."
```

Comment:
```suggestion
            f"Model file does not exist: '{model_path}'. If the model doesn't exist, download it using the 'ilab model download' command. Please verify the model path in 'config.yaml' or pass '--model-path'."
```

I would not abruptly exit 1 but rather raise an exception so the caller can catch and exit properly. Please use a try/except here. Thanks!